### DISCUSSION TITLE
Latest releases appear with Pre-release label

### DISCUSSION THREAD

**Discussion by berkerol** [rancher/rke2#6867] [Q&A] [answered]
Hi,

If you look at all recently released versions (v1.31.1, v1.30.5, v1.29.9, v1.28.14), they all are released with Pre-release label. Is that intentional or is it a release bug?

Thank you.

**Comment by brandond** [comment, marked as answer]
Releases are left in pre-release until our release process is complete, and QA has signed off on all steps being successful. There is more to our process than just creating tags in the repo.

**Reply by berkerol** [reply]
Interesting. But release and its assets stay the same, right? Thanks.

**Reply by brandond** [reply]
Yes? It's just a checkbox on the release that we clear when the process is complete.
